__tests__/app.test.js

```diff
@@ -96,6 +96,37 @@ describe("/api/articles", () => {
   });
 });
 
+describe('/api/articles?topics', () => {
+  test("Returns list of articles with correct topic property", () => {
+    const topic = 'mitch'
+    return request(app)
+      .get(`/api/articles?topic=${topic}`)
+      .expect(200)
+      .then(({body}) => {
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: 'mitch',
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          })
+        })
+      })
+  })
+  test("Returns 404 when querying a valid topic with no entries", ()=> {
+    const topic = 'paper'
+    return request(app)
+      .get(`/api/articles?topic=${topic}`)
+      .expect(404)
+      .then(({body}) => {
+        expect(body).toEqual({msg: 'No articles found'})
+      })
+  })
```

Comment:
^^^ related to above. Please add a 404 topic not found for a topic which doesn't exist e.g. topic=banana